calm/flow-example/account-creation/account-creation-flow.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/draft/2024-09/meta/flow.json",
+  "flows": {
```

Comment:
With the suggested change of bringing flows array to the top level you should lose the surrounding flows object.